mmaction/core/evaluation/eval_hooks.py

```diff
@@ -222,8 +291,8 @@ def __init__(self,
                  dataloader,
                  start=None,
                  interval=1,
-                 save_best=True,
-                 key_indicator='top1_acc',
+                 by_epoch=True,
+                 save_best=None,
```

Comment:
maybe we should rename `save_best`, since it represents the metrics for the best pth.

and assert whether it in the supported measure type.

Reply:
`save_best` serves as both boolean and str, it can provide whether to save best and save which best. Besides, it can be asserted in `_init_rule` as `key_indicator`.